app/javascript/gobierto_investments/webapp/mixins/mataro.conf.js

```diff
@@ -0,0 +1,206 @@
+import axios from "axios";
+
+export default {
```

Comment:
Is there an issue to extract this custom JS to somewhere outside the base code?

Reply:
Nope. Only you must update properly in the `mixin/common.js` file the path where is located this configuration.

But, this file is a technical debt at the moment. It shouldn't exist, it currently plays the middleware role, but such configuration must be provided from backoffice, and the property `itemSpecialConfiguration` must be included in the data endpoint itself. These changes require ETL, API and front-end modifications.
There was an issue regarding this, but it's outdated: https://github.com/PopulateTools/issues/issues/796 (I'm gonna update it with these points)

Reply:
Regarding your question, yes, we can place it somewhere else, but does it worth? Do we proceed? If so, where do we place it?